backend/internal/notification/otpservice.go

```diff
@@ -275,7 +275,7 @@ func (s *otpService) createSessionToken(sessionData common.OTPSessionData) (stri
 	validityPeriod := (sessionData.ExpiryTime - time.Now().UnixMilli()) / 1000
 	jwtConfig := config.GetThunderRuntime().Config.OAuth.JWT
 
-	token, _, err := s.jwtSvc.GenerateJWT("otp", jwtConfig.Issuer, jwtConfig.Issuer, validityPeriod, claims)
+	token, _, err := s.jwtSvc.GenerateJWT("otp-svc", "otp-svc", jwtConfig.Issuer, validityPeriod, claims)
```

Comment:
shouldn't we use `aud` as application here?

Reply:
Better to rethink on the `sub` too..

Reply:
This is the session token intended to be used by the OTP service to verify the previous session. Hence added OTP service as the audience

Reply:
Will add the otp id as the subject as discussed offline. Since we don't have an id for otps, will adrress this separately